ansible_runner/config/_base.py

```diff
@@ -479,20 +485,17 @@ def wrap_args_for_containerization(self, args, execution_mode, cmdline_args):
                                             dst_mount_path="/runner/artifacts",
                                             labels=":Z")
 
-            # Mount the entire private_data_dir
-            # custom show paths inside private_data_dir do not make sense
-            self._update_volume_mount_paths(new_args,
-                                            "{}".format(self.private_data_dir),
-                                            dst_mount_path="/runner",
-                                            labels=":Z")
         else:
             subdir_path = os.path.join(self.private_data_dir, 'artifacts')
             if not os.path.exists(subdir_path):
                 os.mkdir(subdir_path, 0o700)
 
-            # Mount the entire private_data_dir
-            # custom show paths inside private_data_dir do not make sense
-            self._update_volume_mount_paths(new_args, "{}".format(self.private_data_dir), dst_mount_path="/runner", labels=":Z")
+        # Mount the entire private_data_dir
+        # custom show paths inside private_data_dir do not make sense
+        self._update_volume_mount_paths(new_args, "{}".format(self.private_data_dir), dst_mount_path="/runner", labels=":Z")
+
+        # Mount the stdout callback plugin from the ansible-runner code base
+        self._update_volume_mount_paths(new_args, "{}".format(get_plugin_dir()), dst_mount_path="/home/runner/.ansible/plugins", labels=":Z")
```

Comment:
My biggest open question for other people is - where should the mount be? Options include:
 - `/home/runner/.ansible/plugins` (this)
 - `/home/runner/.ansible/plugins/callback`
 - `/home/runner/.ansible/plugins/callback/awx_display.py` (single file mount)
 - `/usr/share/ansible/plugins/callback`
 - `/usr/share/ansible/plugins/callback/awx_display.py` (single file mount)
 - `/usr/share/ansible/collections/ansible_collections/<collection>/<name>/plugins/callback/awx_display.py`

The current option doesn't seem to have any benefits compared to the two options after it. After all, consolidating it into a single file was _so that_ I could do the single file mount. I haven't made this change yet because I wanted to do a bit of a survey of preferences first.

Related to this, but still separate, is the question of where it should go within the code base. In this PR, I nested it in the "callback" folder, but to be able to collection-ify this in the future, I see the wisdom in nesting it in "plugins/callback" folders

Reply:
I seem to have ruled out the single file mount options for now. Doing this results in permission errors like:

> ERROR! Unexpected Exception, this is probably a bug: [Errno 13] Permission denied: '/home/runner/.ansible/plugins/callback/awx_display.py'

Maybe this could be fixed with changes to the Dockerfile, but doing so would partially defeat the point.

With my latest push, the mount location is at `/home/runner/.ansible/plugins/callback`.

Reply:
thanks for detailing your thought process. I like `/home/runner/.ansible/plugins/callback` as the result